auth_backends/backends.py

```diff
@@ -113,14 +90,9 @@ def start(self):
         #    before session cleanup. True if user was logged in, False otherwise.
         set_custom_attribute('session_cleanup.logout_required', user_authenticated)
 
-        if user_authenticated and ENABLE_OAUTH_SESSION_CLEANUP.is_enabled():
+        if user_authenticated:
             existing_username = getattr(request.user, 'username', 'unknown')
 
-            # .. custom_attribute_name: session_cleanup.logged_out_username
-            # .. custom_attribute_description: Records the username that was logged out
-            #    during session cleanup for tracking and debugging purposes.
-            set_custom_attribute('session_cleanup.logged_out_username', existing_username)
```

Comment:
Let's also remove `session_cleanup.logout_performed`. I think `session_cleanup.logout_required` is close enough and has a simpler implementation.